src/DataFixtures/AppFixtures.php

```diff
@@ -15,7 +16,7 @@ public function load(ObjectManager $manager): void
     {
         $faker = Factory::create();
         
-        foreach (range(1, 10) as $i) {
+        foreach (range(1, 10) as $_i) {
```

Comment:
psalm doesn't like unused variables. Putting underscore here, tells psalm to ignore it.